### PR TITLE
Fixes some PHP notices & adds stricter equals checking in MutationType.php

### PR DESCRIPTION
**Pull request by NiklasBr**

### CODE REVIEW

**Comment by NiklasBr** [comment]
```
PHP Notice: Undefined index: create in GraphQL/Mutation/MutationType.php on line 666
PHP Notice: Undefined index: update in GraphQL/Mutation/MutationType.php on line 598
PHP Notice: Undefined index: create in GraphQL/Mutation/MutationType.php on line 400
PHP Notice: Undefined index: update in GraphQL/Mutation/MutationType.php on line 444
PHP Notice: Undefined index: delete in GraphQL/Mutation/MutationType.php on line 768
PHP Notice: Undefined index: delete in GraphQL/Mutation/MutationType.php on line 533
```

File: src/GraphQL/Mutation/MutationType.php [hunk 1]
```diff
@@ -397,7 +397,7 @@ public function buildCreateFolderMutation($type, &$config, $context)
         $configuration = $context['configuration'];
         $entities = $configuration->getSpecialEntities();
 
-        if ($entities[$type . "_folder"]["create"]) {
+        if (isset($entities[$type . "_folder"]["create"])) {
```
Comment (weisswurstkanone) [inline src/GraphQL/Mutation/MutationType.php]: Are you sure? what if the value is FALSE?

Like
                    "asset_folder" => [
                        "id" => "asset_folder",
                        "read" => TRUE,
                        "create" => TRUE,
                        "update" => FALSE,
                        "delete" => TRUE
                    ],

Comment (NiklasBr) [reply src/GraphQL/Mutation/MutationType.php]: Good catch, I'll investigate

**Comment by weisswurstkanone** [comment]
thx!

**Pull request merged**